doclog.config.pl

```diff
@@ -5,10 +5,14 @@
 omit(["ops_and_meta_predicates.pl", "tabling"]).
 learn_pages_source_folder("learn").
 learn_pages_categories(["First steps", "Tutorials"]).
+reif_learn_pages_categories( ["First steps", "Tutorials"]).
```

Comment:
This is unnecessary (it doesn't do anything), if you want to use the Tutorials category you don't need to modify anything, just add the page in learn_pages/1 (as you also did).